src/js/components/Select/SelectContainer.js

```diff
@@ -483,7 +471,9 @@ const SelectContainer = forwardRef(
                       plain={!child ? undefined : true}
                       align="start"
                       kind={!child ? 'option' : undefined}
-                      hoverIndicator={!child ? undefined : 'background'}
+                      // applies theme.global.hover.background to the active
+                      // option for mouse and keyboard interactions
+                      activeIndicator={!child ? undefined : 'background'}
```

Comment:
@ericsoderberghp the activeIndicator changes are needed for this functionality only (isolated to select) and I don't think should be a documented prop on button.

Reply:
Retaining support for `theme.global.hover.background` is needed for backwards compatibility

Reply:
https://github.com/grommet/grommet/pull/6243/files#r927865579

Reply:
Let me see if I've got this right ...
We want to display the same visual as hovering but we don't want that to be tied strictly to mouse hovering but also when traversing via the keyboard. We are already setting `active` to follow the keyboard interaction. We just need a way to indicate that we want an indicator for this active state that mirrors the hover indicator.
If I've got that right ...
Thinking about this purely from the context of Button ...
I would expect that we would use `theme.global.active.background` because we want the `active` indicator styling. `hoverIndicator` can be set to `false` disable hover styling. I would expect `activeIndicator` to be set to `false` to disable the `active` styling. I think it's the explicit connection of `active` with `hover` via `getHoverIndicatorStyle()` in StyledButton when `activeIndicator` is set that seems off to me. A Button can be `active` independently of hovering. So, we want to be able to style `active` and `hover` independently.
Some suggestions ...
`<Button activeIndicator="hover" />` would indicate that the `active` indication should match the `hover` indication.
OR
Put the logic for getting the style from the theme in SelectContainer and pass this through to `activeIndicator` with StyledButton just adding it.
OR
Move the logic into SelectOption where it would just check for `active` in the props and then do the `getHoverIndicatorStyle()`
I'm leaning towards this last one at the moment.

Reply:
The last one seems reasonable to me -- that would remove the need to touch Button I think.